kafka/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - kafka
 
+## 2.0.0 / 2019-06-01
+
+* [Changed] Update metric units. See [#3834](https://github.com/DataDog/integrations-core/pull/3834).
```

Comment:
We should include the metric name here

Reply:
For confirmation, should this indeed be a major bump?

Reply:
To me this should not have been a release at all, nothing in the check changed, this is weird to see a major bump 😕

Reply:
I didn't catch that :( But I agree since it wasn't a code change we probably shouldn't have released.